src/P5Functions/touchmouse.js

```diff
@@ -315,6 +315,8 @@ module.exports = function withTouch(p, curElement, attachEventHandler, document,
 
     if (delta && typeof p.mouseScrolled === 'function') {
       p.mouseScrolled();
+      e.preventDefault();
+      e.stopPropagation();
```

Comment:
@Pomax , that's a very strange nuisance request to do w/o proving whether there is any actual diff. to invoke  **stopPropagation()** and/or **preventDefault()** before **mouseScrolled()**. Given those settings will only become relevant once **mouseWheelHandler()** actually returns.